From b9c2a47f051cce2a6597f2a23d8817608dab508f Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 14 Mar 2019 03:55:57 -0700
Subject: [PATCH 0571/1851] dt-bindings: display: xlnx: vpss-csc: Add max width
 and height properties

commit 92f328841e408f80242b0a1a5de5ea4ded61c74c from
https://github.com/Xilinx/linux-xlnx.git

This patch adds xlnx,max-width and xlnx,max-height mandatory dt
properties for the VPSS as CSC. This breaks backward compatibility.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt    | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
index 29190b9d0fb3..cf80d185d429 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,vpss-csc.txt
@@ -17,6 +17,12 @@ Required properties:
 
 - clocks: phandle to IP clock.
 
+- xlnx,max-width: Maximum number of pixels in a line.
+  Valid range from 64 to 8192.
+
+- xlnx,max-height: Maximum number of lines in a frame.
+  Valid range from 64 to 4320.
+
 Example:
 	csc@a0040000 {
 		compatible = "xlnx,vpss-csc";
@@ -24,4 +30,6 @@ Example:
 		reset-gpios = <&gpio 0x0 GPIO_ACTIVE_LOW>;
 		xlnx,video-width = <8>;
 		clocks = <&misc_clk_0>;
+		xlnx,max-width = <3840>;
+		xlnx,max-height = <2160>;
 	}
-- 
2.31.1

